src/test/java/core/basesyntax/service/handler/OperationHandlerTest.java

```diff
@@ -0,0 +1,49 @@
+package core.basesyntax.service.handler;
```

Comment:
```suggestion
package core.basesyntax.service.handler;
```
здається цей клас має бути в іншому місці)